actions/workflows/hosts.last_checkin.yaml

```diff
@@ -0,0 +1,23 @@
+version: 1.0
+
+description: Workflow to list all hosts that haven't reported 
+
+input:
+  - server
+  - username
+  - password
+  - search
+  - per_page
+
+tasks:
+  task1:
+    action: foreman.hosts.index server=<% ctx(server) %> username=<% ctx(username) %> password=<% ctx(password) %> search=<% ctx(search) %> per_page=<% ctx(per_page) %>
```

Comment:
instead of having tons of parameters all on the action line you can do:
```
action: foreman.hosts.index
input:
  server: "{{ ctx().server }}"
  username: "{{ ctx().username }}"
... etc
```